jobwatcher/plugins/torque.py

```diff
@@ -9,12 +9,16 @@ def get_required_nodes(instance_properties):
     status = ['Q']
     _output = run_command(command, {})
     output = _output.split("\n")[5:]
-    nodes = 0
+    slots_requested = 0
+    nodes_requested = []
     for line in output:
         line_arr = line.split()
         if len(line_arr) >= 10 and line_arr[9] in status:
-            nodes += int(line_arr[5])
-    return nodes
+            slots_requested += int(line_arr[5])
```

Comment:
It should be [6] here and [5] the line below.
Here the expected output of the `qstat -a` command:
```

ip-10-0-235-170.eu-west-1.compute.internal:
                                                                                  Req'd       Req'd       Elap
Job ID                  Username    Queue    Jobname          SessID  NDS   TSK   Memory      Time    S   Time
----------------------- ----------- -------- ---------------- ------ ----- ------ --------- --------- - ---------
8.ip-10-0-235-170.eu-w  centos      batch    job120.sh           --      1      1       --   01:00:00 Q       --
```

Reply:
Done.